platforms/documentation/docs/src/docs/userguide/best-practices/bp_general.adoc

```diff
@@ -93,3 +93,66 @@ include::sample[dir="snippets/bestPractices/avoidInternal/groovy",files="build.g
 
 === Tags
 `#upgrades`
+
+[[favor_composite_builds]]
+== Favor Composite Builds over `buildSrc`
```

Comment:
Should we mention this is for plugins in the title?

For Plugins Favor Composite Builds over `buildSrc`

Reply:
I meant for it to be broader than plugins, as there are other types of non-plugin build-logic you might want to include: like libraries containing task types (or types used by task types) you might need to use in buildscripts.  This is probably much less common though.

Reply:
OK, I'll add that clarification to the title.